lamindb/integrations/_lightning.py

```diff
@@ -0,0 +1,77 @@
+"""PyTorch Lightning integrations.
+
+.. autosummary::
+    :toctree: .
+
+    Callback
+"""
+
+from pathlib import Path
+from typing import Any
+
+import lightning as pl
+from lightning.pytorch import LightningModule, Trainer
+
+import lamindb as ln
+
+
+class Callback(pl.Callback):
+    """Saves PyTorch Lightning model checkpoints to LaminDB after each training epoch.
+
+    Creates version families of artifacts for given `key` (relative file path).
+
+    Args:
+        path: Path to the checkpoint
+        key: Artifact key in LaminDB storage
```

Comment:
Because this can be confusing in several ways, I'd remove "in LaminDB storage".

There should be periods at each of the sentence.